util/test/checkCopyrights.bash

```diff
@@ -18,7 +18,7 @@
 #  Make*
 #
 
-todate=`date -d 2022-01-01 "+%m%d"`
+todate=`date "+%m%d"`
 cond=`date -d 2022-01-07 "+%m%d"`
```

Comment:
This year's still hardcoded, though.  I'm guessing you added the year because of year wrap-around stuff?

Reply:
Yes so date accepts a full date like that, but %m%d cuts off the year so it only compares 0107 to the current date

Reply:
Okay, sounds good!  I'm okay with this going in today

Reply:
Though, might be good to comment it :)